src/wp-includes/class-wp-theme-json.php

```diff
@@ -2713,56 +2714,81 @@ private static function get_block_nodes( $theme_json, $selectors = array(), $opt
 		}
 
 		foreach ( $theme_json['styles']['blocks'] as $name => $node ) {
-			$selector = null;
-			if ( isset( $selectors[ $name ]['selector'] ) ) {
-				$selector = $selectors[ $name ]['selector'];
-			}
 
-			$duotone_selector = null;
-			if ( isset( $selectors[ $name ]['duotone'] ) ) {
-				$duotone_selector = $selectors[ $name ]['duotone'];
-			}
+			$include_node_paths_only = $options['include_node_paths_only'] ?? false;
```

Comment:
This could be moved outside of the `foreach` loop since that option doesn't change